### PR TITLE
Adding a new doc on how to redirect users after login

### PR DESCRIPTION
**Pull request by angelaLazar**
https://trello.com/c/7YU6x0jt/378-add-doc-about-how-to-return-users-to-t
he-page-where-they-came-from

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/protocols/index.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ https://${account.namespace}/authorize/?
 
 `connection` is the only parameter that is Auth0 specific. The rest you will find in the spec. Its purpose is to instruct Auth0 where to send the user to authenticate. If you omit it, the user will be redirected to a hosted login page.
 
-> A note on `state`. This is an optional parameter, but we __strongly__ recommend you use it as it mitigates [CSRF attacks](http://en.wikipedia.org/wiki/Cross-site_request_forgery).
+> A note on `state`. This is an optional parameter, but we __strongly__ recommend you use it as it mitigates CSRF attacks. [Click here to learn more.](/protocols/oauth-state)
```
Comment (mpaktiti) [inline articles/protocols/index.md]: @angelaLazar I'm working on [a PR that will delete this file](https://github.com/auth0/docs/pull/2689/files#diff-d54af3a7abfd13bd7ad090be88e9f0adL1). This is the latest doc on Authorization Code Grant: https://auth0.com/docs/api-auth/tutorials/authorization-code-grant

Maybe you want to undo this change and instead apply it for the new doc instead?

Comment (angelaLazar) [reply articles/protocols/index.md]: @mpaktiti thanks for letting me know. Fixed in latest commit.
